addNewGuitar.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+require_once 'functions.php';
+
+$db = getDB();
+
+$message = '<p>I want to add a new guitar for you!</p>';
+$brandInDB = checkBrands($db);
```

Comment:
These function calls are causing errors, need to be protected when the page is opened for the first time